module/VuFind/config/module.config.php

```diff
@@ -677,6 +692,7 @@
 
 // Define dynamic routes -- controller => [route name => action]
 $dynamicRoutes = [
+    'Feedback' => ['form' => 'Form/[:id]'],
```

Comment:
Is this route used? I haven't had a chance to do testing, but just curious if we need both the feedback-form route above and this dynamic route, or if one is enough.

Reply:
It seems that this is needed for non-default forms, for example /Feedback/Form/AcquisitionRequest

Reply:
See my changes in f84ec63 -- this version seems to work for me, and it's a bit simpler. Please let me know if I've somehow broken something!

Reply:
...and commit 7d58181c7914d7cd4ca32a3a035dc1b2c7237524 is even simpler yet!